### DISCUSSION TITLE
I don't think this works anymore

### DISCUSSION THREAD

**Discussion by ghost** [UltimaHoarder/UltimaScraper#1928] [Q&A]
<img width="342" alt="Screenshot 2022-09-30 at 16 09 17" src="https://user-images.githubusercontent.com/106033128/193301036-5780c905-cedf-4891-8641-8583b76deb02.png">

After running the scraper it just exits the terminal

**Comment by dnsamw** [comment]
u need  python 3.9.x it doesn,t work for 3.10..
